test/unit/pcv/lido/EthLidoPCVDeposit.test.ts

```diff
@@ -72,8 +79,18 @@ describe('EthLidoPCVDeposit', function () {
   });
 
   describe('setMaximumSlippage()', function () {
+    beforeEach(async function () {
+      // grant PCV_MINOR_PARAM_ROLE to governorAddress if it doesn't have it
+      const role = '0x46797b318ce8c2d83979760ef100a5c0fdb980de4b574d6142ce4d0afce307ed';
```

Comment:
Not for now but `ethers.utils.id('PCV_MINOR_ROLE')` is a nice self-documenting way of doing this